targets/ath79-generic

```diff
@@ -89,6 +89,13 @@ device('enterasys-ws-ap3705', 'enterasys_ws-ap3705i', {
 
 -- GL.iNet
 
+device('gl.inet-6416a-v1', 'glinet_6416', {
```

Comment:
According to the DTS, the image name should be "gl.inet-6416".

Reply:
I really need to read up on the first field. Until now I'm just blindly guessing.
Is the rule just `lowercase of the dts' modelname and replace spaces with dashes`?

Reply:
resolved and rebased to master in edc8d0a433571b84d026320be6a5263ba9b18b45

```diff
-device('gl.inet-6416a-v1', 'glinet_6416', {
+device('gl.inet-6416', 'glinet_6416', {
```

Reply:
> I really need to read up on the first field. Until now I'm just blindly guessing. Is the rule just `lowercase of the dts' modelname and replace spaces with dashes`?

The current algorithm is: String is lowercased. All characters that aren't alphanumeric, `.` or `+` are replaced with dashes. Then double dashes and dashes at the end are removed. "-." is replaced with "-". This is implemented here: https://github.com/freifunk-gluon/packages/blob/master/libs/libplatforminfo/src/common.h#L63

Reply:
Thanks, that helped a lot.